### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by MathiasTELITSINE**
Hi,
Contributed some translations, would be glad to help the translation effort if that's possible.
Tell me if anything is wrong.
Thanks.
Mathias

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -77,19 +77,19 @@ msgstr "Qu'est-ce que la possession ?"
 
 #: src/SUMMARY.md:29
 msgid "References and Snapshots"
-msgstr ""
+msgstr "Références et snapshots"
 
 #: src/SUMMARY.md:31
 msgid "Using Structs to Structure Related Data"
-msgstr ""
+msgstr "Utiliser des structures pour structurer des données liées"
```
Comment (enitrat) [inline po/fr.po:84]: > Utiliser des structures structurer des données liées

Can you replace `structures` by `structs` in these titles pls?

File: po/fr.po [hunk 2]
```diff
@@ -306,10 +306,19 @@ msgid ""
 "subset of CASM. The point of Sierra is to ensure your CASM will always be "
 "provable, even when the computation fails."
 msgstr ""
+"Cairo est un langage de programmation conçu pour un processeur virtuel du même nom. "
+"L'aspect unique de ce processeur est qu'il n'a pas été créé pour les contraintes physiques de notre monde, " 
+"mais pour des contraintes cryptographiques, ce qui lui permet de prouver efficacement l'exécution de n'importe quel programme s'exécutant sur celui-ci. "
+"Cela signifie que vous pouvez effectuer des opérations longues sur une machine "
```
Comment (enitrat) [inline po/fr.po:312]: > des opérations longues

complexes au lieu de longues

File: po/fr.po [hunk 3]
```diff
@@ -326,17 +335,30 @@ msgid ""
...
 "reduced transaction costs but preserving Ethereum security."
 msgstr ""
+"Cairo permet de calculer des valeurs fiables sur des machines non fiables. "
+"Un cas d'utilisation majeur est Starknet, une solution de scalabilité pour Ethereum. "
+"Ethereum est une plateforme de blockchain décentralisée qui permet la création d'applications décentralisées "
+"où chaque interaction entre un utilisateur et une application décentralisée est vérifiée par tous les participants. "
+"Starknet est une seconde couche construite au-dessus d'Ethereum. "
+"Au lieu de demander à tous les participants du réseau de vérifier toutes les interactions des utilisateurs, "
+"seul un nœud, appelé le "prover", exécute les programmes et génère des preuves que les calculs ont été effectués correctement. "
+"Ces preuves sont ensuite vérifiées par un contrat intelligent Ethereum, "
```
Comment (enitrat) [inline po/fr.po:345]: > contrat intelligent

smart contract déployé sur

File: po/fr.po [hunk 4]
```diff
@@ -326,17 +335,30 @@ msgid ""
...
+"ce qui nécessite une puissance de calcul considérablement inférieure par rapport à l'exécution des interactions elles-mêmes. "
```
Comment (enitrat) [inline po/fr.po:346]: > interactions

transactions

**Review by enitrat — COMMENTED** [review]
Thanks, I just noted a few terms that could be changed. Otherwise, amazing work 👏

**Comment by MathiasTELITSINE** [comment]
> Thanks, I just noted a few terms that could be changed. Otherwise, amazing work 👏

Thanks a lot for this really fast review. Done the corrections, thanks for telling me i'll implement those on the following translations.

**Review by enitrat — APPROVED** [review]
LGTM!

**Pull request merged**